### ISSUE TITLE
Apply screen: Org Name and EIN autofill strings have some duplication.

### ISSUE THREAD

**Issue by kfogel** [OpenTechStrategies/cgap-poc-demo#41]
I love how the autofills (for name, address, email, phone, etc) show the organization name off to the right, with EIN in parentheses next to it, so the applicant can double-check that they've got the right org.

However, for the Org Name field itself, and for the EIN field itself, some of that extra information off to the right is, of course, now redundant.  For the Org Name field, we should show only EIN in parens as the extra info, and for the EIN field we should correspondingly show the org name but not the EIN in parens on the right.

This bugfix is a "nice to have", not a "must have".  If I get time, I may try to fix it in parallel while you work.  But @slifty if you know off the top of your head how to fix it, please go for it.
